### PR TITLE
Add license file generation

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* Add license file generation
* Readme badge
* CI check
* SPDX header

### CODE REVIEW

File: REUSE.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+# SPDX-License-Identifier: AGPL-3.0-or-later
+version = 1
+SPDX-PackageName = "flow_notifications"
```
Comment (Pytal) [inline REUSE.toml:4]:
```suggestion
SPDX-PackageName = "user_migration"
```
😉

**Review by Pytal — APPROVED** [review]
🐘

File: REUSE.toml (hunk 1 above)
Comment (AndyScherzinger) [reply REUSE.toml:4]: Whoops, fixed/merged via #808

**Pull request merged**